pkg/logs/launchers/file/launcher.go

```diff
@@ -284,12 +284,9 @@ func (s *Launcher) launchTailers(source *sources.LogSource) {
 			continue
 		}
 
-		mode, _ := config.TailingModeFromString(source.Config.TailingMode)
+		mode, isSet := config.TailingModeFromString(source.Config.TailingMode)
 
-		if source.Config.Identifier != "" {
-			// only sources generated from a service discovery will contain a config identifier,
-			// in which case we want to collect all logs.
-			// FIXME: better detect a source that has been generated from a service discovery.
+		if !isSet {
```

Comment:
I think we want this to be 
```suggestion
		if !isSet && source.Config.Identifier != "" {
```

otherwise all tailers will be forced to start from the beginning by default. Currently (as defined in `TailingModeFromString`) tailers wills tart from the end by default.